core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -222,10 +222,7 @@ public int compare(String o1, String o2) {
         }
       });
     }
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("Children nodes: {}", validChildren.size());
-      validChildren.forEach(c -> LOG.debug("- {}", c));
-    }
+    LOG.trace("Children nodes (size: {}): {}", validChildren.size(), validChildren);
```

Comment:
Keeping the original format may be easier to read in the logs.

Reply:
> Keeping the original format may be easier to read in the logs.

In the extremely rare cases somebody is going to enable trace logging to debug something, I think it'd be pretty easy to take the string form of the List type and convert it to a more readable form, if they really need to. Also, there's no guarantee about ordering of log messages in the logger framework, so these can easily be split up by other messages or ordered in strange ways by the time they make it to a log file. Keeping it in one line eliminates any uncertainty with regard to that.

Reply:
Here is what the logs look like now:
<pre>
2021-02-10T09:18:15,632 [zookeeper.ZooLock] DEBUG: validating and sorting children at path /accumulo/1791d843-ecd6-4a3b-85d8-19cfbeee854c/tservers/ip-10-113-12-25:10000
2021-02-10T09:18:15,632 [zookeeper.ZooLock] DEBUG: Children nodes: 1
2021-02-10T09:18:15,632 [zookeeper.ZooLock] DEBUG: - zlock#01437c1b-09a0-484a-8299-bf6562f9ae26#0000000000
2021-02-10T09:18:15,633 [zookeeper.ZooLock] DEBUG: validating and sorting children at path /accumulo/1791d843-ecd6-4a3b-85d8-19cfbeee854c/tservers/ip-10-113-12-25:10000
2021-02-10T09:18:15,633 [zookeeper.ZooLock] DEBUG: Children nodes: 1
2021-02-10T09:18:15,633 [zookeeper.ZooLock] DEBUG: - zlock#01437c1b-09a0-484a-8299-bf6562f9ae26#0000000000
</pre>

Here is what they look like on one line:
<pre>
2021-02-10T09:29:53,922 [zookeeper.ZooLock] DEBUG: Children nodes (size: 1): [zlock#47cb1017-5a01-4018-b6a2-4ca01a634136#0000000000]
2021-02-10T09:29:53,970 [zookeeper.ZooLock] DEBUG: Children nodes (size: 1): [zlock#850d0675-6d54-43fe-9d1b-7423c3e90998#0000000000]
</pre>

Reply:
FYI I switched the log to DEBUG just for comparison.

Reply:
Yeah, I would expect that most times there is only one child, sometimes two.